crates/adapters/src/static_compile/catalog.rs

```diff
@@ -794,9 +795,9 @@ mod test {
             .unwrap();
         input_stream_handle.flush();
 
-        circuit.step().unwrap();
+        circuit.transaction().unwrap();
```

Comment:
step can be interpreted as a verb, which makes sense.
maybe this should be "transact"?